arcgis-ios-sdk-samples/Scenes/Get Elevation at a point/GetElevationPoint.storyboard

```diff
@@ -0,0 +1,87 @@
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Get Elevation Point View Controller-->
+        <scene sceneID="Qjl-9X-TwF">
+            <objects>
+                <viewController id="Pwy-4F-G4n" userLabel="Get Elevation Point View Controller" customClass="GetElevationPointViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="Yv5-8d-hM0">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="gI1-ae-TH7" userLabel="Scene View" customClass="AGSSceneView">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                                <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                            </view>
+                            <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="TbC-bb-iKU" userLabel="Elevation Point Label">
+                                <rect key="frame" x="207" y="762" width="0.0" height="0.0"/>
+                                <color key="backgroundColor" white="0.33333333333333331" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                        </subviews>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstAttribute="trailing" secondItem="gI1-ae-TH7" secondAttribute="trailing" id="1Bz-ok-tRv"/>
+                            <constraint firstItem="TbC-bb-iKU" firstAttribute="centerX" secondItem="Yv5-8d-hM0" secondAttribute="centerX" id="6lA-g5-dyB"/>
+                            <constraint firstItem="gI1-ae-TH7" firstAttribute="top" secondItem="Yv5-8d-hM0" secondAttribute="top" id="LCg-to-Qb6"/>
```

Comment:
I was experiencing what appeared to be the scene view intercepting touches over the navigation bar (such as tapping the back button), even though it underlaps the navigation bar. That may have been the issue you were describing? In any case, please modify this constraint's second item to be the top of the safe area, which should solve the problem.